.circleci/config.yml

```diff
@@ -83,10 +83,12 @@ jobs:
       - run: yarn run ci
       - run: yarn run example:build:prod
       - run: yarn run example:test --watch=false
+      - run: yarn run example:cypress:ci
```

Comment:
I think we should move the example app ones into a separate job and leave this one for unit tests.

Reply:
I've also moved `yarn run example:build:prod` to the separate job, because we need this in order to run the e2e tests 😄